changelog.md

```diff
@@ -1,3 +1,7 @@
+Pending Next Release
+=========
+- Update to Microsoft.Identity.Abstractions 5.2.0
```

Comment:
Do we like this? Preemptively adding changelog notes? 🤔

Reply:
💖
I like it very much.
you might also want to add the feature you're bringing by this PR :-)?

Reply:
Oh, of course! 😆 Done!